printer_mods/Fiction/Crossrail/readme.md

```diff
@@ -0,0 +1,34 @@
+# Crossrail
+
+~~Crossrail is a railway construction project underway mainly in central London. Its aim is to provide...~~
+
+Crossrail is a printed adapter for RSP-200-24 or LRP-200-24 power supplies and DRP-03 brackets. It alows the power supply to be mounted securely across the rails (or vertically as it's sometimes called).
+
+![](./images/_DSF0728.jpg)
+
+crossrail_slot is the same mount but with a slot to allow 10mm adjustment in either direction
```

Comment:
Probably should read "The alternate part 'crossrail_slot' ...."

Reply:
Changed as suggested